layout.html

```diff
@@ -117,8 +117,8 @@
             {# PAGE CONTENT #}
 	    {% if is_release %}
 	    <div class="wy-alert wy-alert-danger">
-		    This is archived documentation for Zephyr version {{ version }}, and not
```

Comment:
not sure archived is the right word here, maybe "released" vs "development" in master

Reply:
How about this then.

Reply:
this released {{ version }} version, missing the d in released?

Reply:
either way sounds OK to me.  I'll yield to your preference.